<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Календарь посещаемости</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="output">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>411</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="res_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверить</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
